services/voting-node/README.md

```diff
@@ -0,0 +1,59 @@
+# General
```

Comment:
Can we consider adding example files with environment variables and where they need to be placed?